pkg/apis/db/v1alpha1/cassandracluster_types_test.go

```diff
@@ -91,35 +91,6 @@ func TestGetNodesPerRacks_2DC(t *testing.T) {
 
 }
 
-func TestGetNumTokensPerRacks_NoTopo(t *testing.T) {
-	assert := assert.New(t)
-
-	cc := helperInitCluster(t, "cassandracluster-NoTopo.yaml")
-
-	nodesPerRack := cc.NumTokensPerRacks("dc-rack1")
-
-	assert.Equal(int32(256), nodesPerRack)
-
-}
-func TestGetNumTokensPerRacks_2DC(t *testing.T) {
-	assert := assert.New(t)
-
-	cc := helperInitCluster(t, "cassandracluster-2DC.yaml")
-
-	numTokens := cc.NumTokensPerRacks("online-rack1")
-	assert.Equal(int32(200), numTokens)
-
-	numTokens = cc.NumTokensPerRacks("online-rack2")
-	assert.Equal(int32(200), numTokens)
-
-	numTokens = cc.NumTokensPerRacks("stats-rack1")
-	assert.Equal(int32(32), numTokens)
-
-	numTokens = cc.NumTokensPerRacks("toto-toto")
-	assert.Equal(int32(256), numTokens)
-
-}
-
```

Comment:
Since the numToken is now directly in Cassandra's config, there is no way / point to test it through this test ? That's why its only a delete and not a refactor ?